vcd_cli/role.py

```diff
@@ -51,19 +54,48 @@ def list_roles(ctx):
               '--org',
               'org_name',
               required=False,
-              metavar='<org-name>',
+              metavar='<org_name>',
               help='name of the org')
-def list_rights(ctx, role_name, org_name=None):
+def list_rights(ctx, role_name, org_name):
     try:
         client = ctx.obj['client']
-        if (org_name is not None):
-            org_href = client.get_org_by_name(org_name).get('href')
-        else:
-            org_href = ctx.obj['profiles'].get('org_href')
-        org = Org(client, org_href, is_sysadmin(ctx))
+        if org_name is None:
+            org_name = ctx.obj['profiles'].get('org_in_use')
+        org = Org(client, resource=client.get_org_by_name(org_name))
         role_record = org.get_role(role_name)
         role = Role(client, href=role_record.get('href'))
         rights = role.list_rights()
         stdout(rights, ctx)
     except Exception as e:
         stderr(e, ctx)
+
+
+@role.command('create', short_help='Creates role in the specified or current organization')
```

Comment:
Creates a new role in the specified organization (defaults to current organization in use)

Reply:
done